src/txmempool.h

```diff
@@ -716,6 +719,12 @@ class CTxMemPool
         return m_unbroadcast_txids;
     }
 
+    // Returns if a txid is in the unbroadcast set
```

Comment:
This can be doxygen like the neighbouring functions. Perhaps touch up in #18807.

Reply:
done